app/models/share/facebook.rb

```diff
@@ -2,24 +2,7 @@ class Share::Facebook < ActiveRecord::Base
   include Share::Variant
 
   validates :description, :title, presence: true
+  belongs_to :image
```

Comment:
ohh right - I actually forgot why I did this in writing the previous comment. 

the real goal here is that instead of making people upload pictures in all the different forms, and likely storing and processing many copies of the same image, we just use the dropzone to upload all the images related to the page. they're all image objects attached to the page. the facebook share then just chooses which of the page's images its associated with. it makes the ui simpler, unifies image interactions, and it prevents storing duplicate images.